key_management.md

```diff
@@ -5,22 +5,22 @@ css_id: key_management
+On the Image repository, the Timestamp and Snapshot metadata SHOULD be signed using online keys, so that an automated process can instantly generate fresh metadata. There are two options for signing the Timestamp and Snapshot metadata, each with the opposite trade-off from the other. In the first option, the OEM uses online keys, meaning automated processes can automatically renew the Timestamp and Snapshot metadata to indicate that new Targets metadata and/or images are available. However, in this scenario, attackers who have compromised a supplier's key would now have access to the Image repository and can instantly publish malicious images. If these attackers also compromise the Director repository, then they can execute arbitrary software attacks by selecting these malicious images on the Image repository for installation. Such an attack could also facilitate mix-and-match attacks.
 
 In the second option, the OEM uses offline keys to sign Timestamp and Snapshot metadata, which reduces the risk of attackers immediately publishing malicious images. The downside deals with expiration of Timestamp and Snapshot.  If the Timestamp and Snapshot metadata expire relatively quickly, then it is more cumbersome to use offline keys to renew their signatures. However, if a longer expiration time is used, then a man-in-the-middle attacker has more time with which to execute freeze attacks, hence defeating the purpose of the Timestamp role.
 
-The keys to all other roles (Root, Targets, and all delegations) on the Image repository SHOULD be kept offline. This is because these metadata are expected to be updated relatively infrequently and so that a repository compromise does not immediately affect full verification ECUs. It does not matter where an offline key is stored (e.g., in a Hardware Security Module, YubiKey, or a USB stick in a safe deposit box), as long as the key is not accessible from the repository. Each key SHOULD be kept separately from others, so that a compromise of one does not affect them all.
+The keys to all other roles (Root, Targets, and all delegations) on the Image repository SHOULD be kept offline. This is because these metadata are expected to be updated relatively infrequently and so that a repository compromise does not immediately affect full verification ECUs. It does not matter where an offline key is stored (e.g., in a Hardware Security Module, YubiKey, or a USB stick in a safe deposit box), as long as the key is not accessible from the repository. Each key SHOULD be kept separate from others, so that a compromise of one does not affect them all.
```

Comment:
```suggestion
The keys to all other roles (Root, Targets, and all delegations) on the Image repository SHOULD be kept offline. This is because these metadata are expected to be updated relatively infrequently and so that a repository compromise does not immediately affect full verification ECUs. It does not matter where an offline key is stored (e.g., in a Hardware Security Module, YubiKey, or a USB stick in a safe deposit box), as long as the key is not accessible from the repository. Each key SHOULD be kept separate from others, so that a compromise of one does not affect them all.
```
```suggestion
The keys to all other roles (Root, Targets, and all delegations, which includes suppliers' keys) on the Image repository SHOULD be kept offline. This is because these metadata are expected to be updated relatively infrequently and so that a repository compromise does not immediately affect full verification ECUs. It does not matter where an offline key is stored (e.g., in a Hardware Security Module, YubiKey, or a USB stick in a safe deposit box), as long as the key is not accessible from the repository. Each key SHOULD be kept separate from others, so that a compromise of one does not affect them all.
```